magento-research/pwa-studio
# feat(seo): add robots.txt static file. Fixes #658.

## Description

Add a simple `robots.txt` file to Venia for SEO. Put it in the `packages/venia-concept/static` directory and adjust the UPWARD conditional for the `/static` DirectoryResolver so it includes `robots.txt`.

## Related Issue





Closes #658 .

## Verification Steps

Visit the URL `/robots.txt`. You should see:

```txt
User-agent: *
Allow: /
```

This is a generic `robots.txt` directive that permits all crawlers to crawl all pages.

## How Have YOU Tested this?
Ensured that the robots.txt file loaded in the browser.

## Proposed Labels for Change Type/Package
`venia-concept`




## Checklist:
